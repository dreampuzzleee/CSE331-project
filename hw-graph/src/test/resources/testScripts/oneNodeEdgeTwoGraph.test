# Tests graph creation/viewing with two graphs ('A' and 'B') with one node and one edge.

# Create a graph
CreateGraph A
CreateGraph B

# Add a node
AddNode A n1

# Add an edge
AddEdge A n1 n1 e1

# Print all nodes in the graph
ListNodes A
ListNodes B

# Print all child nodes of n1 with outgoing edge
ListChildren A n1