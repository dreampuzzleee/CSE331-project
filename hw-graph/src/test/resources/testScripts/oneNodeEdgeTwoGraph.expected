# Tests graph creation/viewing with two graphs ('A' and 'B') with one node and one edge.

# Create a graph
created graph A
created graph B

# Add a node
added node n1 to A

# Add an edge
added edge e1 from n1 to n1 in A

# Print all nodes in the graph
A contains: n1
B contains:

# Print all child nodes of n1 with outgoing edge
the children of n1 in A are: n1(e1)